cg/apps/crunchy/sbatch.py

```diff
@@ -29,6 +29,22 @@
 fi
 """
 
+STAGE_FASTQ_TO_SPRING_COMMANDS = """
+mkdir -p {tmp_dir}
+{conda_run} crunchy -t 12 --tmp-dir {tmp_dir} compress fastq -f {fastq_first} -s {fastq_second} \
```

Comment:
So the command run is `<path/to/conda/binary> run --name <name_of_environment> crunchy blabla...`? I found it quite hard to puzzle together but I'm not sure if it can be much clearer.

Reply:
Yes, much more like a container command. I think this will do for now.